t3/README.md

```diff
@@ -0,0 +1,197 @@
+# T3 Track
```

Comment:
Very nice! Many parts for the "for participants" could be shared among the three tracks. Let's do this after merging this PR.